src/sunlinsol/CMakeLists.txt

```
@@ -28,6 +28,19 @@ if(BUILD_SUNLINSOL_CUSOLVERSP)
   add_subdirectory(cusolversp)
 endif()
 
+if(BUILD_SUNLINSOL_GINKGO)
+  install(CODE "MESSAGE(\"\nInstall SUNLINSOL_GINKGO\n\")")
+  message(STATUS "Added BUILD_SUNLINSOL_GINKGO module")
+  add_library(sundials_sunlinsolginko INTERFACE)
+  target_include_directories(sundials_sunlinsolginko INTERFACE
+    $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+    $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
+    $<BUILD_INTERFACE:${SUNDIALS_SOURCE_DIR}/src/sundials>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+  install(FILES ${PROJECT_SOURCE_DIR}/include/sunlinsol/sunlinsol_ginkgo.hpp
+    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/sunlinsol")
```

Since this does not use `sundials_add_library` we won't have a target in `SUNDIALSConfig.cmake` for the Ginkgo interface but that's probably not necessary since the right include path will get picked up from the package imported target. However, we should update `_SUNDIALS_INSTALLED_COMPONENTS` here so `find_package` can be used with the `COMPONENTS` to check for Ginkgo support. I think the patch below should do it.
```suggestion
    DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/sunlinsol")
  set(_SUNDIALS_INSTALLED_COMPONENTS "sunlinsolginkgo;${_SUNDIALS_INSTALLED_COMPONENTS}" CACHE INTERNAL "" FORCE)
```